android-quick-start/app/src/main/java/com/amazon/location/android/quickstart/MainActivity.kt

```diff
@@ -0,0 +1,127 @@
+package com.amazon.location.android.quickstart
+
+import android.os.Bundle
+import android.util.Log
+import android.view.animation.OvershootInterpolator
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import com.amplifyframework.AmplifyException
+import com.amplifyframework.auth.cognito.AWSCognitoAuthPlugin
+import com.amplifyframework.core.Amplify
+import com.amplifyframework.geo.location.AWSLocationGeoPlugin
+import com.amplifyframework.geo.location.models.AmazonLocationPlace
+import com.amplifyframework.geo.maplibre.view.MapLibreView
+import com.amplifyframework.geo.maplibre.view.support.fadeIn
+import com.amplifyframework.geo.maplibre.view.support.fadeOut
+import com.amplifyframework.geo.models.Coordinates
+import com.amplifyframework.geo.options.GeoSearchByCoordinatesOptions
+import com.mapbox.mapboxsdk.camera.CameraPosition
+import com.mapbox.mapboxsdk.geometry.LatLng
+import com.mapbox.mapboxsdk.maps.MapboxMap
+import kotlin.math.abs
+
+class MainActivity : AppCompatActivity() {
+    private val mapView by lazy {
+        findViewById<MapLibreView>(R.id.mapView)
+    }
+
+    private val descriptionView by lazy {
+        findViewById<TextView>(R.id.description_text_view)
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        initAmplify()
```

Comment:
Since Amplify can only be configured once at runtime, it's best to add the plugins and configure Amplify in an application class to avoid edge cases where the onCreate for an activity might be called more than once.

Reply:
That's a good callout. I will move the initialization to an Application class.

Reply:
I have moved initialization of Amplify to a dedicated class named `AndroidQuickstartApp` :)